cms/djangoapps/contentstore/toggles.py

```diff
@@ -81,3 +81,60 @@ def exam_setting_view_enabled():
     Returns a boolean if proctoring exam setting mfe view is enabled.
     """
     return ENABLE_EXAM_SETTINGS_HTML_VIEW.is_enabled()
+
+
+# .. toggle_name: new_core_editors.use_new_text_editor
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: This flag enables the use of the new core text xblock editor
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2021-12-1
+# .. toggle_target_removal_date: 2022-1-30
+# .. toggle_tickets: TNL-9306
+# .. toggle_warnings:
+ENABLE_NEW_TEXT_EDITOR_FLAG = WaffleFlag('new_core_editors.use_new_text_editor', __name__)
+
+
+def use_new_text_editor():
+    """
+    Returns a boolean = true if new text editor is enabled
+    """
+    return ENABLE_NEW_TEXT_EDITOR_FLAG.is_enabled()
```

Comment:
Question: We've enabled/disabled other features such as the new courseware MFE on a course-by-course basis. The way this toggle is currently written, the new text editor will be either globally on or off. Is that the plan?

Reply:
Nothing was scoped to include "turn on/off for x,y,z course." If you think that should be a requirement, lmk.

Reply:
We decided yesterday in the parking lot with @marcotuts input that there's no need to enable/disable by course. So it's fine this way.